service/controller/appcatalog/v1/key/key.go

```diff
@@ -13,6 +13,10 @@ func AppCatalogTitle(customObject v1alpha1.AppCatalog) string {
 	return customObject.Spec.Title
 }
 
+func CatalogStorageURL(customObject v1alpha1.AppCatalog) string {
```

Comment:
Same here we should use `customResource` here to align with the new naming in `fmt`.